packages/core/src/scp-cf/environment-accessor.ts

```diff
@@ -41,7 +40,7 @@ export function getDestinationBasicCredentials(): BasicCredentials {
  * @returns The 'destination' credentials object or null if it does not exist.
  */
 export function getDestinationServiceCredentials(): any {
-  return head(getDestinationServiceCredentialsList());
+  return getDestinationServiceCredentialsList()[0];
```

Comment:
I would prefer to have a `first` or `head` funciton in the util to get the element. I know it is trivial, but if we use it for last I want to symmetric on first.

Reply:
Hmm ok ;)